kale/task.py

```diff
@@ -76,6 +90,10 @@ def __init__(self, message_body=None, *args, **kwargs):
 
         self._dequeued()
 
+        if settings.is_multiprocess_mode() and not Task._task_process_pool:
+            # The worker must be initialized after everything is loaded.
+            Task._task_process_pool = multiprocessing.Pool(1)
```

Comment:
I'd recommend moving all of this into a class getter that memoizes the threadpool.